src/Components/ErrorBoundary.tsx

```diff
@@ -13,11 +13,13 @@ interface Props {
 
 interface State {
   asyncChunkLoadError: boolean
+  genericError: boolean
```

Comment:
These changes were rebased from https://github.com/artsy/reaction/pull/3272; once that's merged this can be rebased from master and will vanish from PR.